packages/openwrt/README.md

```diff
@@ -1,62 +1,48 @@
 
-1. Download and extract the SDK toolchain for your device. The toolchain
-   must match the *exact* OpenWRT version installed in your device. Toolchain
-   for official OpenWRT images can be downloaded from https://downloads.openwrt.org
+   Note: The SDK must match the **exact OpenWrt version and architecture** installed in your device.
 
-2. Build the toolchain: run `make menuconfig`, save the configuration, then
-   run `make` to build the cross compiling tools
+   For details, see [\[OpenWrt Wiki\] Using the SDK](https://openwrt.org/docs/guide-developer/toolchain/using_the_sdk).
 
-3. Download the feeds with `./scripts/feeds update -a`
+3. Update feeds
 
-## Compilation
+   ```bash
+   ./scripts/feeds update -a && ./scripts/feeds install -a
+   ```
 
-These instructions are for building the current checked out version of the
-n2n source  (The generally used OpenWRT alternative is to download a tar.gz
-file of a specific n2n version, but that is not as suitable for development
-or local builds)
+4. Clone the n2n source code
 
-You need both the openwrt repository and the n2n repository checked out
-for this.  In these instructions, we assume that `openwrt` is the directory
-where your openwrt checkout is located and `n2n` is the directory for
-the n2n repository.
+   ```bash
+   git clone https://github.com/ntop/n2n packages/n2n
+   ```
 
-```
-git clone https://github.com/ntop/n2n n2n
-N2N_PKG_VERSION=$(n2n/scripts/version.sh)
-export N2N_PKG_VERSION
+5. Build n2n
 
-cp -r n2n/packages/openwrt openwrt/package/n2n
+   ```bash
+   make defconfig
+   make package/n2n/packages/openwrt/compile -j$(nproc) V=s
+   ```
 
-cd openwrt
-make menuconfig # select Network -> VPN -> n2n-edge and n2n-supernode
-make package/n2n/clean V=s
-make package/n2n/prepare USE_SOURCE_DIR=../n2n V=s
-make package/n2n/compile V=s
-```
+6. Install
 
-If everything went fine, two ipk will be generated, one for the n2n-edge
-and the other for n2n-supernode. They can be found with `find . -name "n2n*.ipk"`,
-copied to the target device, and installed with `opkg install`.
+   If everything went fine, four ipk will be generated.
 
-The github action described in `.github/workflows/openwrt.yml` implements
-an automated version of the above steps.
+   Generally speaking, you just need the n2n-edge and n2n-supernode.
 
-## Configuration
+   They can be found with `find bin/packages -name "n2n*.ipk"`.
 
-The edge node can be started with `/etc/init.d/edge start`.
-Its configuration file is `/etc/n2n/edge.conf`.
+   Copy them to the target device, and install with `opkg install`.
 
-The supernode can be started with `/etc/init.d/supernode start`.
-Its configuration file is `/etc/n2n/supernode.conf`.
+7. Configure
+
+   The edge node can be started with `/etc/init.d/edge start`. Its configuration file is `/etc/n2n/edge.conf`.
+
+   The supernode can be started with `/etc/init.d/supernode start`. Its configuration file is `/etc/n2n/supernode.conf`.
```

Comment:
This is another file that appears to be basically a rewrite.  Some of the changes here do not appear to improve the documentation, they look like they have deleted some explanatory text.  Is there some reasoning behind these changes that you can expand on?

Reply:
Yeah, I just updated the build instructions and some outdated lines were removed. 
"explanatory text" is at OpenWrt docs where I pointed out.